core/mock_test.go

```diff
@@ -371,7 +371,7 @@ func (wg *mockNodeWg) getDone() int64 {
 }
 
 func (wg *mockNodeWg) resetDone() {
-	wg.count = 0
+	atomic.StoreInt64(&wg.count, 0)
```

Comment:
Are we going to use go 1.19? It'd be nice to have things like https://pkg.go.dev/sync/atomic#Int64

Reply:
Not until we bump the version for Edge, and I don't see it happening until 1.19 stabilizes

Reply:
Could you a bit clarify, what do you mean on '1.19 stabilizes'?

Reply:
We can't bump the package version here to `1.19`, since Edge is still on `1.18`. I believe we've had an internal discussion on some Slack channel regarding the version bump, and decided to stick with `1.18` for now as people were reporting problems with `1.19`

Reply:
What kind of problems? THe only problem I know is with linters for 1.18 then https://github.com/golangci/golangci-lint/issues/2649

Reply:
This is the [thread](https://mvpworkshop.slack.com/archives/C02EZ0KUGF4/p1664271009129269) I'm talking about. Regardless, we need to stick to `1.18` for now until we bump the go version elsewhere

Reply:
I don't believe I have access to it.
Anyway I was surprised with 1.19 issues because we use it in Bor already - https://github.com/maticnetwork/bor/blob/develop/go.mod#L3

Reply:
Resolving this convo for merging